src/base/LruMap.h

```diff
@@ -14,21 +14,34 @@
 #include <list>
 #include <map>
 
-template <class EntryValue, size_t EntryCost = sizeof(EntryValue)> class LruMap
+template<class E>
```

Comment:
Please use `EntryValue` instead of `E` to better illustrate the purpose if this function.